src/core/plugin/index.ts

```diff
@@ -135,7 +135,8 @@ export type RuntimeContributions<C extends {} = any> = {
     create: (req: Express.Request) => C
   }
   nexus?: {
-    plugins: NexusConfig['plugins']
+    typegenAutoConfig?: NexusConfig['typegenAutoConfig']
+    plugins?: NexusConfig['plugins']
```

Comment:
Used to be required because it was the only field. Now there are two fields. And a plugin might only need one of them.